synadm/cli/__init__.py

```diff
@@ -41,80 +41,75 @@ class APIClient(api.Synapse_admin):
     }
 
     CONFIG = {
-        "user": None,
-        "token": None,
-        "base_url": None,
+        "user": "",
+        "token": "",
+        "base_url": "",
```

Comment:
admin_path should be admin_api_path so it reflects the name of the `synadm config` option